editor/editor_inspector.cpp

```diff
@@ -1972,6 +2002,53 @@ Size2 EditorInspectorSection::get_minimum_size() const {
 	return ms;
 }
 
+EditorInspector *EditorInspectorSection::_get_parent_inspector() const {
+	Node *parent = get_parent();
+	while (parent) {
+		EditorInspector *ei = Object::cast_to<EditorInspector>(parent);
+		if (ei) {
+			return ei;
+		}
+		parent = parent->get_parent();
+	}
+	return nullptr;
+}
+
+Control *EditorInspectorSection::make_custom_tooltip(const String &p_text) const {
+	if (!checkable) {
+		return Container::make_custom_tooltip(p_text);
+	}
+
+	String symbol;
+	String prologue;
+
+	if (object->has_method("_get_property_warning")) {
+		const String custom_warning = object->call("_get_property_warning", related_enable_property);
+		if (!custom_warning.is_empty()) {
+			prologue = "[b][color=" + get_theme_color(SNAME("warning_color")).to_html(false) + "]" + custom_warning + "[/color][/b]";
```

Comment:
The group color should be yellow too:
![image](https://github.com/user-attachments/assets/3a39ee03-5c54-4d7a-86de-9144fd8d5709)
Also looks like the warning color does not work here for some reason.

Compare with regular property:
![image](https://github.com/user-attachments/assets/f942665f-6d9b-4778-9009-9230cdde6089)